front/src/components/MatchOverview.vue

```diff
@@ -0,0 +1,84 @@
+<template>
+<div class="match">
+  <div class="row">
+    <span v-if="played_at">
+      Played at <time :datetime="played_at">{{ playedAtFormatted }}</time>
+    </span>
+    <span v-else>
+      Played at an unknown time
+    </span>
+  </div>
+  <div class="row">
+    <strong class="left-col">{{ teamOne.name }}</strong>
+    <span class="middle-col">vs</span>
+    <strong class="right-col">{{ teamTwo.name }}</strong>
+  </div>
+  <div class="row">
+    <span class="left-col">{{ scoreOne }}</span>
+    <span class="middle-col">:</span>
+    <span class="right-col">{{ scoreTwo }}</span>
+  </div>
+  <div class="row">
+    <router-link :to="{ name: 'match', params: { id } }">See more</router-link>
+  </div>
+</div>
+</template>
+
+<script>
+import moment from 'moment'
+import Team from './Team'
+
+export default {
+  data () {
+    return {
+      editable: false,
+    }
+  },
+  components: {
+    Team,
+  },
+  props: {
+    id: Number,
+    played_at: String,
```

Comment:
Sometimes you use camelCase and sometimes underscores. What about sticking to one convention?